operator/controller/src/test/resources/k8s/examples/tls/insecure-traffic-with_tls.apicurioregistry3.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: registry.apicur.io/v1
+kind: ApicurioRegistry3
+metadata:
+  name: simple
+spec:
+  app:
+    env:
+    ingress:
+      host: simple-app.apps.cluster.example
+    tls:
+      insecureRequests: enabled
+      truststoreSecretRef:
+        name: truststore
+        key: truststore
+      truststorePasswordSecretRef:
+        name: truststore
+        key: password
+      keystoreSecretRef:
+        name: keystore
+        key: keystore
+      keystorePasswordSecretRef:
+        name: keystore
+        key: password
+  ui:
+    ingress:
+      host: simple-ui.apps.cluster.example
+    env:
+      - name: REGISTRY_API_URL
+        value: https://simple-app.apps.cluster.example/apis/registry/v3
```

Comment:
I think we should add a note to this example that the feature is intended for TLS inside of the cluster for now, and they would have to create their own https reencrypt Ingress.